QuickTour/QuickTour.tex

```diff
@@ -408,23 +408,27 @@ \section{Workspaces and transcripts}
 (The \menu{(d)} in the menu item \menu{do it (d)} tells you that the keyboard shortcut to \emph{do it} is \short{d}.
 More on this in the next section!)
 
-%%{SBESqueakPicture writeTo: './figures/Doit.png' frame: [:morphsRect | morphsRect withBottom: morphsRect bottom - 150] using: [:helper |
-%%  | workspace |
-%%  ToolBuilder open: Transcript.
-%%  Transcript show: 'hello world'; cr.
-%%  helper foregroundWindow bounds: (0 @ 120 extent: 450 @ 230).
-%%  
-%%  workspace := ToolBuilder open: Workspace new.
-%%  workspace position: 200 @ 100.
-%%  
-%%  helper type: 'Transcript show: ''hello world''; cr.' into: workspace.
-%%  helper keyStroke: workspace key: $a modifiers: #(cmd).
-%%  
-%%  helper click: workspace at: workspace topCenter + (0 @ 30) buttons: #(right).
-%%  helper hand releaseMouseFocus.
-%%  helper mouseEnter: (helper menuItemMorphLabeled: 'do it (d)' in: helper world).
-%%  helper closeBalloons.
-%%]}
+\iffalse
+figures/Doit.png created via
+SBESqueakPicture
+	writeTo: './figures/Doit.png'
+	frame: [:morphsRect | morphsRect withBottom: morphsRect bottom - 150]
+	using: [:helper | | workspace |
+	ToolBuilder open: Transcript.
+	Transcript show: 'hello world'; cr.
+	helper foregroundWindow bounds: (0 @ 120 extent: 450 @ 230).
+
+	workspace := ToolBuilder open: Workspace new.
+	workspace position: 200 @ 100.
+
+	helper type: 'Transcript show: ''hello world''; cr.' into: workspace.
+	helper keyStroke: workspace key: $a modifiers: #(cmd).
+
+	helper click: workspace at: workspace topCenter + (0 @ 30) buttons: #(right).
+	helper hand releaseMouseFocus.
+	helper mouseEnter: (helper menuItemMorphLabeled: 'do it (d)' in: helper world).
+	helper closeBalloons]
+\fi
```

Comment:
```suggestion
\begin{ExecuteSmalltalkScript}
SBESqueakPicture writeTo: './figures/Doit.png' frame: [:morphsRect | morphsRect withBottom: morphsRect bottom - 150] using: [:helper | | workspace |
	ToolBuilder open: Transcript.
	Transcript show: 'hello world'; cr.
	helper foregroundWindow bounds: (0 @ 120 extent: 450 @ 230).
	
	workspace := ToolBuilder open: Workspace new.
	workspace position: 200 @ 100.
	
	helper type: 'Transcript show: ''hello world''; cr.' into: workspace.
	helper keyStroke: workspace key: $a modifiers: #(cmd).
	helper click: workspace at: workspace topCenter + (0 @ 30) buttons: #(right).
	helper hand releaseMouseFocus.
	helper mouseEnter: (helper menuItemMorphLabeled: 'do it (d)' in: helper world).
	helper closeBalloons
]
\end{ExecuteSmalltalkScript}
```

Provided that I did not miss any special intentions from your side, we already have a better mechanism for this, which I have named in the absence of a better idea the `ExecuteSmalltalkScript` environment. See also all its other "senders": `grep ExecuteSmalltalkScript **/*.tex`.

Also, note that the scripts are already parsed automatically as part of the CI pipeline (`.travis.yml`), see `SBEScriptBuilder class >> #collectScriptsFromStream:` and `SBEScriptBuilder class >> #collectScriptsClassicFromStream:` (the latter parses the old `%%{SBESqueakPicture` notation which should eventually be deprecated). By the way, the figure building process must not be confused with the `@TEST` code sections, see [this overview here](https://github.com/hpi-swa-lab/SqueakByExample-english#build-process) and last but not least also the Makefile target `listings`, which is a third unrelated mechanism. :-)